File: SeungjaeLim/PGS/codingtest_kit/hash/완주하지 못한 선수.py
def solution(participant, completion):
    pdict = {}
    for p in participant:
        if p in pdict:
            pdict[p] += 1
        else:
            pdict[p] = 1
    for c in completion:
        pdict[c] -= 1
    for k,v in pdict.items():
        if v != 0:
            return k
        

File: ise-yen/PGS/완주하지 못한 선수.cpp
#include <string>
#include <vector>
#include <unordered_map>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    
    unordered_map<string, int> um;
    for(int i = 0; i < participant.size(); i++){
        auto it = um.find(participant[i]);
        if(it == um.end()){
            um.insert({participant[i], 1});
        }
        else{
            um[participant[i]]++;
        }
    }
    
    for(int i = 0; i < completion.size(); i++){
        auto it = um.find(completion[i]);
        if(it != um.end()){
            um[completion[i]]--;
        }
    }
    
    for(auto it : um){
        if(it.second != 0){
            answer = it.first;
        }
    }
    
    return answer;
}
